test/e2e/universal/healthcheck/panic.go

```diff
@@ -36,7 +36,7 @@ conf:
 	dp := func(idx int) string {
 		return fmt.Sprintf(`
 type: Dataplane
-mesh: default
+mesh: hc-panic
```

Comment:
Maybe the mesh name can be put in a package level var? Or can we create a `newDataplane() string` function?

Reply:
for `newDataplane()` - We need to have resource builders. This is also a small subproject on its own.
I'll create an issue and let's prioritize this for eng backlog.

I think that the Mesh should be on `Describe` level. In one package you may have many Describes and I'd like them to run concurrently.

For this one I can parameterize the yamls

Reply:
https://github.com/kumahq/kuma/issues/4133